crumbs-fix.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# This script is intended as a temporary fix for the missing ">" breadcrumb symbols in the file browser that only occurs when we build the frontend in production mode.
+# CARTAvis/carta-frontend issue #698
+# It seems to be a sass-loader bug. The problem may eventually go away, but meanwhile this script will fix it. 
+# This script will be run automatically at the last step of "npm run build".
+
+if [ "$(uname)" == "Darwin" ]; then
```

Comment:
Also, where do these values come from, and how likely are they to change in the short term? They seem to be the same in my install. It may be safer to extract them from the files than to hardcode them:

```
wrong=`grep -Po "breadcrumbs>li:after\{.*?\}" main.*.css | grep -Po "d='.*?'"`
right=`grep -Po ".bp3-breadcrumbs > li::after\{.*?\}" main.*.css.map | grep -Po "d='.*?'"`

(later...)

$sedcmdhere "s/$wrong/$right/w somewhere" badfile # note double quotes
```

But this is just more fragile regex; if the values are probably going to be constant, it's overkill. This was tested on Linux; I'm not 100% sure if it needs any modifications for BSD grep.

Reply:
Thanks for the suggestions @confluence Yes. This patch is only intended to be temporary so probably not worth spending too much time to make it too perfect.